packages/generator/src/edmx-parser/edmx-file-reader.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { readEdmxFile } from './edmx-file-reader';
+import {
+  parseActionImport,
+  parseActions,
+  parseEntitySets,
+  parseEntityType,
+  parseEnumTypes,
+  parseFunctionImports,
+  parseFunctions
+} from './v4';
+import { parseComplexTypesBase } from './common';
+
+describe('edmx-file-reader', () => {
+  it('does not fail for multiple schema entries in the edmx file', () => {
```

Comment:
Maybe remove the one from `packages/generator/src/edmx-parser/v2/edmx-parser.spec.ts`?

Reply:
oh yes good point. I copied it but forgot to remove it.